content/examples/shortcodes/ref-relref/en.md

```diff
@@ -23,3 +24,20 @@ The result will be just like a link. But the difference is that with ref/relref
 
 [Buttons example]({{< relref "buttons/index.md" >}})
 
+Relref can also be used for anchor links:
+
+```
+[Anchor links example]({{<ref "#anchor-header" >}})
```

Comment:
This example renders oddly, negating the point that I think you're trying to make (that `relref` *must* be used, and **not** just bare fragments inside a Markdown link):

![image](https://github.com/user-attachments/assets/dcbce452-3828-4fd1-be71-df7ce8b63de6)